bindings/swift/Sources/COpenDAL/module.modulemap

```diff
@@ -0,0 +1,23 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+module COpenDAL [system] {
```

Comment:
It this the naming style in swift?

Reply:
Yep, module names with `C-` prefix is like `-sys` crate in Rust. Those modules only contain raw C APIs.

Ref: https://github.com/apple/swift-system/blob/main/Package.swift#LL23C18-L23C18